src/jdk.incubator.foreign/share/classes/jdk/incubator/foreign/CLinker.java

```diff
@@ -112,12 +113,25 @@
      * @return a linker for this system.
      * @throws IllegalAccessError if the runtime property {@code foreign.restricted} is not set to either
      * {@code permit}, {@code warn} or {@code debug} (the default value is set to {@code deny}).
+     * @throws UnsupportedOperationException if called when running on an unsupported platform.
      */
     static CLinker getInstance() {
         Utils.checkRestrictedAccess("CLinker.getInstance");
         return SharedUtils.getSystemLinker();
     }
 
+    /**
```

Comment:
Not sure about this - I wouldn't want to draw more attention on this than it deserves

Reply:
Removed now